setup.py

```diff
@@ -102,6 +105,9 @@ def get_extras_require():
             'torchvision'
         ] if sys.version_info[:2] < (3, 8) else []),
         'testing': [
+            # TODO(Yanase): Remove pillow from dependencies after torchvision includes the
+            # following change: https://github.com/pytorch/vision/pull/1501.
+            'pillow<7',
```

Comment:
A nit but it used to be sorted alphabetically so we might better stick to it.

Reply:
You're right. I'll fix it.

Reply:
Fixed: https://github.com/optuna/optuna/pull/827/commits/8786c234c3318ca18d6269348e5b1233bfade596.

Reply:
Thanks!